MatrixSDKTests/MXCryptoBackupTests.m

```diff
@@ -515,9 +515,9 @@ - (void)testTrustForKeyBackupVersion
 // - Alice and Bob have messages in a room
 // - Alice has cross-signing enabled
 // - Alice creates a backup
+// - Modify the backup to remove all signatures other than the MSK so we know that the MSK alone is enough for the device to be trusted
 // - Check the returned MXKeyBackupVersion is trusted
-// -> It must be trusted by 2 entities
-// -> Trusted by her device
+// -> It must be trusted by 1 entities
 // -> Trusted by her MSK
 - (void)testCrossSigningMSKTrustForKeyBackupVersion
```

Comment:
Repurposed an existing test. Based on the name it seems to have been wanting to test MSKTrust, but because the valid device signature was marking the device usable it was hiding the bug that I'm fixing with this PR. Now the only signature on the backup is the MSK.

Reply:
The new test looks good, however it might be better to create it as copy and leave `testCrossSigningMSKTrustForKeyBackupVersion` as is, otherwise we loose the test case for getting a device signature (`XCTAssertEqual(keyBackupVersionTrust.signatures.count, 2)`)